### PR TITLE
Applies OpenStack project cookiecutter

### PR DESCRIPTION
**Pull request by claudiubelu**
All OpenStack projects have a certain project template which
they follow, and it can easily applied via cookiecutter.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -53,3 +53,6 @@ ChangeLog
 *~
 .*.swp
 .*sw?
+
+# Files created by releasenotes build
+releasenotes/build
```
Comment (alexcoman) [inline .gitignore]: Please add a new line.

File: releasenotes/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+# The encoding of source files.
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'hnv Release Notes'
+copyright = u'2016, OpenStack Foundation'
```
Comment (alexcoman) [inline releasenotes/source/conf.py]: Shouldn't be 2017 ?

File: setup.cfg [hunk 3]
```diff
@@ -34,12 +33,28 @@ oslo.config.opts =
...
+
+[extract_messages]
+keywords = _ gettext ngettext l_ lazy_gettext
+mapping_file = babel.cfg
+output_file = hnv/locale/hnv.pot
+
+[build_releasenotes]
+all_files = 1
+build-dir = releasenotes/build
+source-dir = releasenotes/source
```
Comment (alexcoman) [inline setup.cfg]: Please add a new line.

File: releasenotes/source/conf.py (hunk 2 above)
Comment (claudiubelu) [reply releasenotes/source/conf.py]: Done.

File: .gitignore (hunk 1 above)
Comment (claudiubelu) [reply .gitignore]: Done.

File: setup.cfg (hunk 3 above)
Comment (claudiubelu) [reply setup.cfg]: Done.

**Comment by alexcoman** [comment]
Thank you @bclau !

**Pull request merged**